### ISSUE TITLE
WCAG: Text spacing

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#26]
## Description
We need to test if it is possible for the user to change different variables of the texts in the apps and not loose any functionality or content

The user should be able to set all of the following style properties:

```
- Line height (line spacing) to at least 1.5 times the font size;
- Spacing following paragraphs to at least 2 times the font size;
- Letter spacing (tracking) to at least 0.12 times the font size;
- Word spacing to at least 0.16 times the font size.
```
[Source](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)

## Acceptance criteria
> Describe criteria here (i.e. What is allowed/not allowed (negative tesing), validations, error messages and warnings etc.)

## Specification tasks

- [ ] Test is if this is possible
  - [ ] If no: How to fix it

**Comment by Febakke** [comment]
Tested and found no errors except some visual bugs on our date picker. Will create a separate issue for date picker
